cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/ProgramScheduleStoreDataset.java

```diff
@@ -530,6 +532,9 @@ private ProgramScheduleMeta extractMetaFromRow(ScheduleId scheduleId, Row row) {
       String triggerKey = Schedulers.triggerKeyForPartition(((PartitionTrigger) trigger).getDataset());
       return Collections.singletonList(triggerKey);
     }
+    if (trigger instanceof ProgramStatusTrigger) {
+      return Collections.singletonList(((ProgramStatusTrigger) trigger).getProgramId().toString());
```

Comment:
yes, we can include that. this is temporary and will be replaced by calls to `trigger.getTriggerKeys()` method